bin/lesson8/main.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:dart_learning_journey/lesson8/aop/aspect_oriented.dart';
+import 'package:dart_learning_journey/lesson8/index.dart';
+
+import 'package:talker/talker.dart';
+
+void main() {
+  final Talker talker = Talker(
+      settings: TalkerSettings(
+          maxHistoryItems: 2000,
+          colors: {TalkerLogType.verbose.key: AnsiPen()..magenta(), TalkerLogType.debug.key: AnsiPen()..cyan()}));
+
+  final AspectOriented lesson8 = AspectOriented(talker);
```

Comment:
```suggestion
  final lesson8 = AspectOriented(talker);
```

This version is totally fine, but in practice we usually take advantage of Dart’s type inference:
https://dart.dev/language/type-system#type-inference